Fix: remove loading file section, fix dark mode

StrongMonkey opened the pull request:
- Drop `loading file` section
- Fix the knowledge details page in dark mode

ryanhopperlowe reviewed (changes requested):
sorry to do this, but we can't use tailwind builtin colors. Please only use colors listed in `tailwind.config.js` and `tailwind.css`

StrongMonkey commented:
yeah @ryanhopperlowe you are right, I was using some inner div which set bd to white. I removed that and everything seems to work fine. Changed to remove the extra div.

StrongMonkey commented on ui/admin/app/components/knowledge/KnowledgeSourceDetail.tsx:
```diff
@@ -313,760 +313,723 @@ const KnowledgeSourceDetail: FC<KnowledgeSourceDetailProps> = ({
     return (
         <Dialog open={isOpen} onOpenChange={onOpenChange}>
-            <DialogContent className=" h-[80vh] w-[80vw] max-w-none max-h-none">
-                <div
```
The only change is the div className moved to DialogContent.

ryanhopperlowe commented:
> yeah @ryanhopperlowe you are right, I was using some inner div which set bd to white. I removed that and everything seems to work fine. Changed to remove the extra div.

Sweet! thanks for fixing it!

StrongMonkey commented on ui/admin/app/components/knowledge/KnowledgeSourceDetail.tsx:
```diff
@@ -313,14 +313,14 @@ const KnowledgeSourceDetail: FC<KnowledgeSourceDetailProps> = ({
+            <DialogContent className="h-[80vh] w-[80vw] max-w-none max-h-none dark:bg-gray-900">
                 <div
-                    className=" sticky top-0 bg-white z-10 overflow-y-auto mr-2"
+                    className="sticky top-0 bg-white dark:bg-gray-900 z-10 overflow-y-auto mr-2"
```
yeah, this is side effect of using too much ai 😄
